examples/06_benchmarks/movielens.ipynb

```diff
@@ -72,28 +72,39 @@
    "execution_count": 1,
```

Comment:
<p>@SahitiCheguru here did you use the same VM as it was before? can you please review the write up?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1758/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
I am using Standard NC6 (6 vcpus, 56 GiB memory).